src/components/button/button.stories.mdx

```diff
@@ -35,10 +34,7 @@ Some buttons may not require as much emphasis as others. The `c-button--secondar
 All the above visual styles will appear slightly different within [our dark theme](/?path=/docs/themes--dark). These versions retain their visual emphasis relative to one another while remaining visually distinct from form inputs.
 
 <Canvas>
-  <Story
-    name="Styles (Dark)"
-    parameters={{ themes: { disable: true }, paddings: { disabled: true } }}
-  >
-    {(args) => stylesDarkDemo(args)}
+  <Story name="Styles (Dark)" parameters={{ theme: 't-dark' }}>
```

Comment:
Should this be `dark` or `t-dark`? You used `dark` in `themes.stories.mdx`.

Reply:
It should be `t-dark`, the `dark` was me passing a noun for the demonstrated theme to the theme demo specifically.